### ISSUE TITLE
When I press shift h or h when I want to go back or forward a heading in firefox the history pops up.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2610]
**Reported by meerkat on 2012-08-20 22:37**
When I press shift h or h to go back or forward a heading in firefox the history pops up.

**Comment by bhavyashah** [comment]
I haven't experienced such issues in recent versions of Firefox and NVDA. Time to close? @ehollig

**Comment by fernando-jose-silva** [comment]
I've already tried this on slow computers.
Playgrounds:
Walk a few times with arrows below and press ctrl to interrupt the speech of the nvda.
After repeated for several times pressing the arrow and ctrl to interrupt the speech me that windows or nvda does not recognize that ctrl is loose.
If this occurs by pressing h to navigate through the titles, the history can be opened as long as ctrl is still pressed.
Comments:
I can not reproduce this currently because I got rid of all very slow computers.
This was a mistake that occurred when the computer was especially slow to process something in the background.
The machine in question was an amd c6 with 1ghz and 2gb of ran with windows 10 64bits.
